2021/viz_evaluation.md

```diff
@@ -0,0 +1,70 @@
+# Evaluation of data visualization effectiveness
+ArviZ is a Python and Julia library for exploratory analysis of Bayesian models,
+and includes many [vizualizations](https://arviz-devs.github.io/arviz/examples/index.html)
+to that end.
+
+As ArviZ developers we try to follow the literature on effective data visualization,
+especially around communicating uncertainty, but we might not have time to
+keep up with everything, we might be weighted down by bad practices we still have
+not managed and we might not always have time to update the code when a new
+recommendation or visualization type is made available.
+Probably a combination of all 3 and some extra factors.
+
+Therefore, to ensure ArviZ is an active agent in advocating and popularizing
+effective data visualization practices we budgeted $2000 as part of our GSoD
+2021 application to pay someone external to the team to evaluate the
+plots that are available in ArviZ and their defaults.
+
+## Job description
+We are expecting a report covering the following points:
+
+* Analysis of available and missing visualizations
+
+  Evaluate which visualizations should be present in ArviZ but are not.
+  Each visualization list how it would serve ArviZ's goal and some
+  degree of addition priority, not necessarily a ranking, could be
+  binning missing visualizations in 3-4 high-low priority categories.
+
+* Analysis of current defaults
+
+  Evaluate if ArviZ defaults follow best practices. Write down both things
+  we are doing well (if any) and things we need to improve. Any and
+  all points related to styling, layout, combination of multiple elements
+  can be considered here.
+
+* Analysis of usage advise
+
+  Evaluate the advise included in the ArviZ documentation about
+  using each plotting function. As a first approximation this
+  should be a list of useful references per plot so users can
+  better understand when and how to use each of them.
+
+The report should be completed by June 2022. We are aiming for an
+ArviZ team meeting and/or Bayesian workflow unconference sometime
+in July-August 2022 and want to have this report available to inform
+our roadmap planning.
+
+We emphasize again the need for the report to include rationale for
+all comments and references where appropriate. While the report will
+be written around ArviZ, it will be published here and we expect
+it to be useful to other similar libraries like bayesplot.
+
+Moreover, it could also act as a seed of a centralized resource
+maintained by all Bayesian data viz libraries with advise on
+how and when to use each plot, combining them...
+
+## Job requirements
+* Not being part of the ArviZ team
+* Familiarity with data visualization and uncertainty visualization literature (TODO: any good way
```

Comment:
maybe too high requirements for the money, but  scientific papers and/or previous experience with scientific visualization?

Reply:
we could add that yes, money doesn't worry too much on that end. The job is clear (hopefully) and fixed and imo $2000 is fair for the job asked. 

I think that the more knowledgeable and aware of current literature they are the less time they'll need to write the report, that should account for that hourly rate needing to be be higher?

Reply:
I agree is fair amount, and you have a very good point  expertise level  and hourly rate.